po/pt_BR.po

```diff
@@ -645,10 +648,9 @@ msgstr ""
 "    diffuse ( -h | -? | --help | -v | --version )\n"
 "\n"
 "O Diffuse é uma ferramenta gráfica para unificar e comparar arquivos de texto.\n"
-"É capaz de comparar um número arbitrário de arquivos lado-a-lado e dá aos\n"
-"usuários a possibilidade de ajustar manualmente as linhas com edição directa.\n"
-"Também pode obter revisões de arquivos de repositórios Bazaar, CVS, Darcs, Git,\n"
-"Mercurial, Monotone, RCS, Subversion e SVK para comparação e unificação.\n"
+"É capaz de comparar um número arbitrário de arquivos lado-a-lado e dá a\n"
+"possibilidade de ajustar manualmente as linhas com edição directa.\n"
+"Também pode obter revisões de arquivos de vários VCSs para comparar e unificar.\n
 "\n"
```

Comment:
The CI is failing here and I think the reason is because you have an additional `\n` at the end here.
Could you try this instead?

```suggestion
"Também pode obter revisões de arquivos de vários VCSs para comparar e unificar.\n"
```

Side note: if you feel like it, you can also add yourself to the translators in the [AUTHORS](https://github.com/MightyCreak/diffuse/blob/master/AUTHORS) files :wink:

Reply:
Sorry about that I didn't see you comment, I believe that it'll be ok now

Side reply:
I sort of based this translation on ptpt so I don’t want to take too much credit for my work ;)